### PR TITLE
include aggregate and config discoveries in root

### PR DESCRIPTION
**Pull request by ktoso**
Make sure we publish the new aggregate and config discoveries

Resolves https://github.com/akka/akka-management/issues/226

### CODE REVIEW

**Comment by ktoso** [comment]
FYI @chbatey

**Comment by ktoso** [comment]
I think we can release a 0.16 soon, no need to backrelease in 15 hmm

File: build.sbt [hunk 1]
```diff
@@ -8,12 +8,14 @@ lazy val `akka-management-root` = project
...
     `akka-discovery-aws-api`,
     `akka-discovery-aws-api-async`,
+    `akka-discovery-config`,
     `akka-discovery-consul`,
+    `akka-discovery-dns`,
+    `akka-discovery-kubernetes-api`,
+    `akka-discovery-marathon-api`,
     `akka-management`,
     `bootstrap-joining-demo-aws-api-ec2-tag-based`,
     `bootstrap-joining-demo-aws-api-ecs`,
```
Comment (TimMoore) [inline build.sbt]: Also missing:

- bootstrap-joining-demo-kubernetes-api
- bootstrap-joining-demo-marathon-api-docker

Comment (ktoso) [reply build.sbt]: Thanks added

**Pull request merged**